### PR TITLE
(update) ive upgrade the gameplay features

### PR DESCRIPTION
**Pull request by BillyGoat12**

### CODE REVIEW

File: client/src/components/GameEnv/2DEnv/2DEnv.jsx [hunk 1]
```diff
@@ -1,21 +1,60 @@
-import React from 'react';
+/* eslint-disable array-callback-return */
```
Comment (jstrzesz) [inline client/src/components/GameEnv/2DEnv/2DEnv.jsx]: we should not be using eslint-disable, find a workaround to get your code working and to not bypass the linter

File: client/src/components/GameEnv/2DEnv/card.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```
Comment (jstrzesz) [inline client/src/components/GameEnv/2DEnv/card.jsx]: see previous comment about eslint-disable

File: client/src/components/GameEnv/3DEnv/3DEnv.jsx [hunk 3]
```diff
@@ -1,15 +1,19 @@
-import React, { Suspense, useRef } from 'react';
+/* eslint-disable no-param-reassign */
```
Comment (jstrzesz) [inline client/src/components/GameEnv/3DEnv/3DEnv.jsx]: see previous comments on eslint-disable

File: client/src/components/GameEnv/3DEnv/3DEnv.jsx [hunk 4]
```diff
@@ -52,21 +56,76 @@ function ArWing() {
...
         <OrbitControls />
         <directionalLight intensity={0.5} />
         <ambientLight intensity={0.5} />
-        <spotLight position={[10, 15, 10]} angle={0.9} />
+        <spotLight position={[20, 20, 10]} angle={0.9} />
         <Suspense fallback={<Loading />}>
-          <ArWing />
+          <Table />
+          <Loading />
+          <Cards position={[6, 2, -13]} slot={slots[3]} />
```
Comment (jstrzesz) [inline client/src/components/GameEnv/3DEnv/3DEnv.jsx:113]: are these cards hardcoded in?

**Review by jstrzesz — CHANGES_REQUESTED** [review]

File: client/src/components/GameEnv/3DEnv/3DEnv.jsx (hunk 4 above)
Comment (BillyGoat12) [reply client/src/components/GameEnv/3DEnv/3DEnv.jsx:113]: not really its where the cards will be placed on the table

**Review by jstrzesz — APPROVED** [review]

**Review by BenYoss — APPROVED** [review]

**Pull request merged**